CONTRIBUTING.md

```diff
@@ -43,28 +43,33 @@ labels.
 
 ## Pull Requests
 
-All pull requests go through CircleCI, which makes sure the code doesn't need to
-be reformatted, builds Coda in its various configurations, and runs all the
+Our branching workflow is [documented on the wiki](https://www.notion.so/minaprotocol/Compatible-vs-Develop-branch-management-31c845b2924b4c518740eb9da4514dcc).
+
+All pull requests go through Buildkite CI, which makes sure the code doesn't need to
+be reformatted, builds Mina in its various configurations, and runs all the
 tests.
 
 All pull requests must get _code reviewed_. Anyone can do a code
 review! Check out the [code review
 guide](https://docs.minaprotocol.com/en/developers/code-reviews) for
 what to look for. Just leave comments on the "Files changed" view.
 
-All pull requests must be approved by at least one member of the "core eng"
-team on github.
+All pull requests must be approved by at least one _code owner_ for the
+source code they touch. See the [CODEOWNERS](./CODEOWNERS) file to
+find out who they are.
 
 Maintainers should assign reviewers to pull requests, and tag them with any
-relevant labels.
+relevant labels. You can skip this step by asking reviewers directly
+in the #review-requests channel on slack, if you have access.
```

Comment:
This channel is not public today.

Reply:
This document is meant to be a reference for new developers, who will probably gain access to this channel. I think it's best to leave it in there, but I did change the wording.